### PR TITLE
Make transaction expiration a config

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Makes transaction expiration a config instead of a hard-coded constant

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: eth_worker/eth_src/sql_persistence/interface.py [hunk 1]
```diff
@@ -540,12 +540,10 @@ def set_block_range_status(self, start, end, status, filter_id):
         self.session.commit()
 
 
-    def __init__(self, red, session, first_block_hash, PENDING_TRANSACTION_EXPIRY_SECONDS=30):
```
Comment (enjeyw) [inline eth_worker/eth_src/sql_persistence/interface.py:543]: wondering on the reasoning against passing this in to the object still? Like we do for the gas price in the processor:

```
processor = EthTransactionProcessor(
    ethereum_chain_id=config.ETH_CHAIN_ID,
    gas_price_wei=w3.toWei(config.ETH_GAS_PRICE, 'gwei'),
```

**Review by enjeyw — COMMENTED** [review]
got a comment on structure

File: config.py [hunk 2]
```diff
@@ -270,6 +270,7 @@ def address_from_private_key(private_key):
             private_key = bytes.fromhex(private_key.replace('0x', ''))
         return keys.PrivateKey(private_key).public_key.to_checksum_address()
 
+    ETH_PENDING_TRANSACTION_EXPIRY_SECONDS = config_parser['ETHEREUM'].get('transaction_expiry_seconds') or 30
```
Comment (enjeyw) [inline config.py:273]: pretty sure this needs to be converted from string to int for the ini file

**Review by enjeyw — CHANGES_REQUESTED** [review]
parserrrrrror

File: config.py (hunk 2 above)
Comment (michielderoos) [reply config.py]: My bad! Fixed!

Comment (enjeyw) [reply config.py]: them ini files will getcha lol!

**Review by enjeyw — APPROVED** [review]
good to merge after version bumpiboo :)

**Comment by tristanhcole** [comment]
[Make transaction expiration a config](https://trello.com/c/HcYuctnq/659-make-transaction-expiration-a-config)

**Pull request merged**